### ISSUE TITLE
CNN followed by LSTM for Classifying multiple shots of a room

### ISSUE THREAD

**Issue by AntreasAntoniou** [fchollet/keras#1325]
Hello there,

I have multiple photos that show different angles of a specific objects and I was hoping to use a CNN as the feature extractor and an LSTM as the classifier so that I can enter for example a sequence of 40 photos and get as an output the room label. How can I do this with keras? I have a problem with the connection between the LSTM and the CNN. Should I use a time distributed dense? Please let me know if keras can help with this. So that I can go and implement something in tensorflow if not.

**Comment by PiranjaF** [comment]
Look at https://github.com/fchollet/keras/blob/master/examples/imdb_cnn_lstm.py

**Comment by AntreasAntoniou** [comment]
Well I am using a Convolution2D what should I use here:
model.add(Embedding(max_features, embedding_size, input_length=maxlen))
model.add(Dropout(0.25))

Should max_features be width*height*channels?
